CMakeLists.txt

```diff
@@ -308,7 +308,9 @@ else()
   message(STATUS "eventdisplay will not be built, because ROOT has no opengl support.")
 endif()
 
-add_subdirectory(MbsAPI)
+if(BUILD_MBS)
```

Comment:
Let's also declare this option as official build [option](https://cmake.org/cmake/help/latest/command/option.html) somewhere before this line.

Reply:
hm, seeing this, let's also rename it to `BUILD_MBSAPI` to match the component name.

Reply:
It's already an official `option`:
https://github.com/FairRootGroup/FairRoot/blob/dev/CMakeLists.txt#L295

So should we rename it, despite it already being "established"?
* Con: The name is already there, in theory people could already know about it and use `-DBUILD_MBS=ON` so be sure, they get this.
* Pro: It was broken anyway. (at least the OFF setting)

I don't have an opinion on this.

Reply:
> So should we rename it, despite it already being "established"?

No, in that case, let's leave it. Sry, didn't remember this.